### PR TITLE
Issue 2054 newsletter

### PR DESCRIPTION
**Pull request by jenniferisasi**
*add the reason for your pull request here*

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [ ] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [ ] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by jenniferisasi** [comment]
Looks like the issue with a URL in a lesson on Markdown leads to a possible need to edit a paragraph in there... will open a ticket to evaluate.

**Comment by jenniferisasi** [comment]
Note to self: when the issue with the URL is fixed and I can update, fix the typo in "acknowledgements"

File: _posts/2021-04-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+title: First Newsletter in 2021
+authors: 
+- Jennifer Isasi
+layout: post
+categories: posts
+---
+
+<img src="/images/blog/ph-banner-4lang.png" alt="Banner of PH with the logos of the four journals" title="Programming Historian"/>  
+
+We are already three months into 2021 and it is time to recap on what *Programming Historian* has been working on. No hemos parado de recibir lecciones originales. Nous avons édité la première leçon en français.  E continuamos trabalhando muito para trazer mais lições em português.
```
Comment (mariechristineb) [inline _posts/2021-04-01-newsletter.md]: 'première leçon _originale_ en français'?

File: _posts/2021-04-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- [Análisis de sentimientos en R con 'syuzhet'](https://programminghistorian.org/es/lecciones/analisis-de-sentimientos-r) de Jennifer Isasi te enseña una forma de obtener y analizar datos sobre las emociones y los sentimientos de una narración. 
+- [Installation de bibliothèques Python avec pip](https://programminghistorian.org/fr/lecons/installation-modules-python-pip) par Fred Gibbs et traduit par Lucas Terriel, il explique l’une des méthodes les plus courantes utilisant pip.
+
+- [Calibrer des âges radiocarbone avec R](https://programminghistorian.org/fr/lecons/calibration-radiocarbone-avec-r) par Nicolas Frerebeau et Brice Lebrun c'est a première leçon originale en français! Cette leçon explique pas à pas comment réaliser la calibration d’un ensemble de dates et comment explorer et présenter les résultats avec R.
```
Comment (mariechristineb) [inline _posts/2021-04-01-newsletter.md]: I would say '[...] Lebrun _est la_ première leçon originale [...]' or '[...] Lebrun _est notre_ première leçon originale [...]'

File: _posts/2021-04-01-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+### Collaboration 
+
+In order for us to be able to better address the needs of our community, Adam Crymble is working with the Centre for Data, Culture and Society, the UCL Centre for Digital Humanities, and the Digital Humanities program at the University of the Andes to build knowledge about how different institutions are using *Programming Historian* in teaching settings. This will allow us to share experiences within our IPP network and cater to pedagogical needs they find in teaching digital methods. 
+
+## Nouveaux membres/novos membros
+
+Notre comité de rédaction continue d’évoluer et de croître dans plusieurs directions. Nous avons recruté trois nouveaux rédacteurs pour l’édition en francais de la revue. Veuillez vous joindre à nous pour accueillir Antoine Henry de Lille University, Gwenaëlle Patat da Maison des Sciences de l'Homme en Bretagne et Matthias Gille Levenson de l'École Normale Supérieure de Lyon. Bienvenue à toutes et à tous!
```
Comment (mariechristineb) [inline _posts/2021-04-01-newsletter.md]: Université de Lille (instead of 'Lille University')

Comment (mariechristineb) [reply _posts/2021-04-01-newsletter.md]: Gwenaëlle Patat _de la_ Maison [...]

File: _posts/2021-04-01-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+We will address the issues that we can fix ourselves and seek help for those for which  we are lacking expertise on.  
+
+## Acknoledgments
+
+A year ago, and before leaving the team, Matthew Lincoln wrote a blogpost about one of the ways in which one can [produce a complex multilingual static site using Jekyll and GitHub pages](https://matthewlincoln.net/2020/03/01/multilingual-jekyll.html). The post was nominated for the Best DH Blog Post or Series of Posts category in the Digital Humanities Awards 2020. Unfortunately, it didn't win, but the nomination showed how important this resource is for the DH community. And congratulations to [the winners](http://dhawards.org/dhawards2020/results/)!
+
+Jennifer Isasi, *PH* comms manager and editor, was nominated and elected to be part of the 2021-2025 Executive Council for [The Association for Computers and the Humanities](https://ach.org). ¡Felicidades, Jennifer! And we would also like to congratulate Kim Gallon and Lorena Gauthereau, who will also part of the Executive Council.
```
Comment (mariechristineb) [inline _posts/2021-04-01-newsletter.md]: 'will also _be_ part'?

**Review by mariechristineb — COMMENTED** [review]
@jenniferisasi, I just added my comments in the review

**Comment by jenniferisasi** [comment]
Since MSU global passed already, I will update that section of the newsletter given it got stuck by another PR.

**Review by ZoeLeBlanc — APPROVED** [review]
All of these look good to me 👍🏽  and sorry for the added files from gh-pages but had to happen at some point I guess 😅

**Pull request merged**